### PR TITLE
Add tests for pre-existing code for user

### PR DESCRIPTION
**Pull request by PaulDebevec**
##PR Description:

Add testing for pre-existing user query functionality, ie. user index and user show. All tests are passing.

##Please include a summary of the change:

Add tests to user/query_testing. 

##If it is a new feature, describe the feature and the iteration of the project spec it addresses:

##Non-new feature. PR is for added testing.

##Type of change
Please delete options that are not relevant.

[x] I have performed a self-review of my own code
[x] I have commented my code, particularly in hard-to-understand areas

##Commits:
Add tests for return all users

Add test for returning a single user

Merge master into branch

### CODE REVIEW

File: code/api/src/modules/user/query.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  it("returns all users", async () => {
+    const response = await request(server)
+      .get('/')
+      .send({ query: '{ users { name email } }'})
+      .expect(200)
+
+      expect(response.body.data.users[0]).toEqual({ "name": "The Admin", "email": "admin@crate.com" });
+      expect(response.body.data.users[1]).toEqual({ "name": "The User", "email": "user@crate.com" });
+      expect(response.body.data.users.length).toEqual(2);
```
Comment (Maxwell-Baird) [inline code/api/src/modules/user/query.test.js:29]: We should remember this line of code if add and more users to our seed files

File: code/api/src/modules/user/query.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  })
+
+  it("returns a single user", async () => {
+    const response = await request(server)
+      .get('/')
+      .send({ query: '{ user(id: 1) { name email } }'})
+      .expect(200)
+
+    expect(response.body.data).toBeTruthy();
+    expect(response.body.data.user.name).toEqual("The Admin")
```
Comment (Maxwell-Baird) [inline code/api/src/modules/user/query.test.js:39]: At a later date we should test multiple fields beyond just name

**Review by Maxwell-Baird — APPROVED** [review]
Nice job with the tests and keeping the test straight forward

File: code/api/src/modules/user/query.test.js (hunk 1 above)
Comment (colintalex) [reply code/api/src/modules/user/query.test.js:29]: I have a few tests like this as well, maybe we can rewrite them to not be dependant?

**Review by colintalex — APPROVED** [review]
Good job on the tests

File: code/api/src/modules/user/query.test.js (hunk 1 above)
Comment (PaulDebevec) [reply code/api/src/modules/user/query.test.js:29]: I agree there's a better way to make it more dynamic. I just couldn't figure out which expectation to use in the time period we have.

File: code/api/src/modules/user/query.test.js (hunk 2 above)
Comment (PaulDebevec) [reply code/api/src/modules/user/query.test.js:39]: I concur. I want to make this test more robust all around.

**Pull request merged**